openapi/business/business.yaml

```diff
@@ -208,6 +208,189 @@ paths:
       tags:
         - Customers
         - Options
+    get:
+      summary: List Customers
+      operationId: get-customers
+      responses:
+        '200':
+          description: OK
+          content:
+            application/vnd.api+json:
+              schema:
+                type: object
+                properties:
+                  data:
+                    type: array
+                    items:
+                      $ref: '#/components/schemas/customers'
+                  links:
+                    type: object
+                    properties:
+                      self:
+                        type: string
+                        format: uri
+                      first:
+                        type: string
+                        description: Provides a link back to the first page of results
+                        format: uri
+                      next:
+                        type: string
+                        description: The URI at which the next batch of customers can be gotten from
+                        format: uri
+      description: |-
+        [Status](https://developers.vendasta.com/platform/ZG9jOjEwMTU2NTYy-versioning): `Trusted Tester`
+
+        Produces a list of customers
+      parameters:
+        - schema:
+            type: string
+          in: header
+          name: Authorization
+          description: A Bearer access token to identify the user the app is acting on behalf of. See the Authorization guide for details.
+          required: true
+        - schema:
+            type: string
+          in: query
+          name: 'filter[businesses.id]'
```

Comment:
I think we've been opting for the term `businessLocation` when referencing account groups.

Reply:
Yes. `filter[businessLocation.id]` is what the majority of endpoints use.